docs/src/content/docs/guides/project-structure.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Project Structure
+description: Learn how to organize files in your Starlight project.
+---
+
+This guide will show you how a Starlight project is organized and what different files in your project do. Starlight projects generally follow the same file and directory structure as other Astro projects. See [Astro’s project structure documentation](https://docs.astro.build/en/core-concepts/project-structure/) for more detail.
+
+## Files and directories
+
+- `astro.config.mjs` — The Astro configuration file; includes the Starlight integration and configuration.
+- `src/content/config.ts` — Content collection configuration file; adds Starlight’s frontmatter schemas to your project.
```

Comment:
```suggestion
- `src/content/config.ts` — Content collections configuration file; adds Starlight’s frontmatter schemas to your project.
```
I'm honestly not sure here! Singlular or plural??

Reply:
Eh, could go either way couldn’t it. I’ll go with the instinct that given you bothered to comment on this, that’s a signal it should be plural 😅